src/pages/mentorship.js

```diff
@@ -43,20 +43,23 @@ export default function Home() {
                 Management.
                 <br />
                 <br />
-                If you are interested in contributing your expertise to mentor junior professionals or recent graduates, we would love for you to join us!
+                This year we are collaborating with <a href="https://www.womenwhocode.com/singapore" target="_blank">WomenWhoCode Singapore</a> to make this program even more impactful.
+                There would be 1 - 1 mentoring sessions with a mentor and mentee with matching criterias. Minimum once a month for 3 months.
+                <br />
+                <br />
+                Registration for the TechLadies Mentorship Program 2023 has just opened. Sign up for this year’s Mentorship Program now!
```

Comment:
Do we want to reiterate the collaboration bit? Just curious, not tied to either.

```suggestion
                Registration for the TechLadies x WWCode SG Mentorship Program 2023 has just opened. Sign up for this year’s Mentorship Program now!
```